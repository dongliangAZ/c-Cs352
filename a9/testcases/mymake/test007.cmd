vgopts=$1    # Valgrind options
outfile=$2
errfile=$3
statusfile=$4

BASEDIR="/home/cs352/spring16/Assignments/assg09/testcases/./mymake"

ulimit -t 10
valgrind $vgopts -q ./mymake ${BASEDIR}/mfile5 t2 >${outfile} 2>${errfile}
exitstatus=$?
echo $exitstatus > $statusfile
