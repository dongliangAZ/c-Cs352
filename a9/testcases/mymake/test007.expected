d3
d2
d4
t2
